hendrix/deploy/base.py

```diff
@@ -151,19 +151,24 @@ def addHendrix(self):
         '''
         self.hendrix = HendrixService(
             self.application,
-            port=self.options['http_port'],
             threadpool=self.getThreadPool(),
             resources=self.resources,
             services=self.services,
             loud=self.options['loud']
         )
+        if self.options["https_only"] is not True:
+            self.hendrix.spawn_new_server(self.options['http_port'], HendrixTCPService)
 
     def catalogServers(self, hendrix):
         "collects a list of service names serving on TCP or SSL"
         for service in hendrix.services:
             if isinstance(service, (TCPServer, SSLServer)):
                 self.servers.append(service.name)
 
+    def _listening_message(self):
```

Comment:
Neat. I can imagine a a class-level dict of messages, and a single method to display them.

Reply:
:heavy_check_mark: